domain/src/main/java/edu/nextstep/camp/calculator/Operation.kt

```diff
@@ -0,0 +1,25 @@
+package edu.nextstep.camp.calculator
+
+data class Operation private constructor(
```

Comment:
```suggestion
data class Operation(
```
data class에는 copy 메서드를 통해서 객체를 만들 수 있기 때문에, private constructor를 사용하지 않는 것을 권장해요.
private constructor를 유지하고 싶으시다면, class로 변경하신 후에 equals, hashCode, toString 등 오버라이딩하셔서 구현해주시면 좋을 것 같아요.